src/flows/recommendation_api/corpus_candidate_sets/topics.py

```diff
@@ -14,13 +14,38 @@
 
 # Export approved corpus items by language and recency
 EXPORT_CORPUS_ITEMS_SQL = """
-SELECT 
-    approved_corpus_item_external_id as "ID", 
-    topic as "TOPIC"
-FROM "ANALYTICS"."DBT"."APPROVED_CORPUS_ITEMS"
-WHERE REVIEWED_CORPUS_ITEM_UPDATED_AT >= DATEADD('day', -90, current_timestamp())
-AND TOPIC = %(CORPUS_TOPIC_ID)s
-AND LANGUAGE = 'EN'
+WITH prep as (
+    SELECT 
+        approved_corpus_item_external_id as "ID", 
+        topic as "TOPIC",
+        MAX(reviewed_corpus_item_updated_at) as "REVIEW_TIME" 
+    FROM "ANALYTICS"."DBT"."APPROVED_CORPUS_ITEMS" 
+    WHERE REVIEWED_CORPUS_ITEM_UPDATED_AT >= DATEADD('day', -90, current_timestamp())
+    AND CORPUS_REVIEW_STATUS = 'recommendation'
+    AND TOPIC = %(CORPUS_TOPIC_ID)s
+    AND SCHEDULED_SURFACE_ID = 'NEW_TAB_EN_US'
+    GROUP BY 1, 2
```

Comment:
Remove `max` and `group by`? Same suggestion for other query in the union.
- We should use the latest topic value, in case curators change them.
- There's no need to deduplicate here if we do it at the end.
```suggestion
    SELECT 
        approved_corpus_item_external_id as "ID", 
        topic as "TOPIC",
        reviewed_corpus_item_updated_at as "REVIEW_TIME" 
    FROM "ANALYTICS"."DBT"."APPROVED_CORPUS_ITEMS" 
    WHERE REVIEWED_CORPUS_ITEM_UPDATED_AT >= DATEADD('day', -90, current_timestamp())
    AND CORPUS_REVIEW_STATUS = 'recommendation'
    AND TOPIC = %(CORPUS_TOPIC_ID)s
    AND SCHEDULED_SURFACE_ID = 'NEW_TAB_EN_US'
```